nav2_bringup/bringup/params/nav2_params.yaml

```diff
@@ -254,6 +254,8 @@ planner_server:
 
     GridBased:
       tolerance: 0.5                    # tolerance for planning if unable to reach exact pose, in meters
+      downsample_costmap: true          # whether or not to downsample the map to the specified resolution
+      costmap_resolution: 0.3           # target resolution for the planner grid, in meters
```

Comment:
Does your method make it easy for weird resolutions (0.354) or non-multiples on the original? I'd think you'd want to make this a multiplier, not a resolution, to force users to only have multiples.